cmd/format-coverage.go

```diff
@@ -46,16 +47,28 @@ var formatCoverageCmd = &cobra.Command{
 	Use:   "format-coverage",
 	Short: "Locate, parse, and re-format supported coverage sources.",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if args[0] != "" {
+			logrus.Debugf("coverage path %s", args[0])
+			formatOptions.CoveragePath = args[0]
+		}
 		return runFormatter(formatOptions)
 	},
 }
 
 func runFormatter(formatOptions CoverageFormatter) error {
 	envy.Set("PREFIX", formatOptions.Prefix)
+
 	// if a type is specified use that
 	if formatOptions.InputType != "" {
 		if f, ok := formatterMap[formatOptions.InputType]; ok {
 			logrus.Debugf("using formatter %s", formatOptions.InputType)
+			if formatOptions.CoveragePath != "" {
```

Comment:
This seems to duplicate the `args[0]` conditional again. Would it be possible to write things there so that doesn't have to happen? Something like:

```
if args[0] != ""
  formatOptions.CoveragePath = args[0];
else
  formatOptions.CoveragePath = <infer by type or error>

<downstream can assume it's always present>
```